config.js

```diff
@@ -11,12 +11,142 @@ System.config({
 System.config({
```

Comment:
Is that how you do it in workflow?

The config.js file is the only way to lock down dependency versions (direct and transitive), which you need for reproducible builds, so I think it's best to commit the config.js file as a version locking mechanism. Happy to discuss if people disagree, but it'd be good to have an agreement and then consistency across projects.